code/game/gamemodes/miniantags/guardian/guardian.dm

```diff
@@ -701,6 +702,23 @@
 		to_chat(user, "[failure_message]")
 		used = FALSE
 
+/obj/item/weapon/guardiancreator/afterattack(var/obj/item/I as obj, mob/user as mob, proximity)
+	..()
+	if(!proximity)
+		return
+	if(!refundable)
+		return
+	if(istype(I, /obj/item))
```

Comment:
drop the `, /obj/item` portion, `istype(I)` will work because it's casted as an item